renovate.json

```diff
@@ -1,5 +1,8 @@
 {
   "extends": [
-    "config:base"
-  ]
+    "config:base",
+    ":automergeLinters",
```

Comment:
Per discussion with @adzuci, it sounds like the automerge flags would automatically merge linting and testing package updates without it passing all of the CI checks. Not sure we'd want to do that, because it means that the person who merges after the automerges may be stuck fixing a pile of linting/testing issues on master (and potentially need to roll back their changes if they can't quickly fix forward, esp. if they didn't rebase after the automerge).

Reply:
Also, is there a list of linting / testing libraries this flag affects?

Reply:
I stole this from prospectus and tried to figure out what was considered a linter or tester library, but unfortunately couldn't see anything with the word lint in https://github.com/edx/frontend-app-payment/blob/master/package.json
https://docs.renovatebot.com/presets-default/#automergelinters

If we checked the second to last box here it would not auto merge PRs with failing tests:
https://github.com/edx/frontend-app-payment/settings/branch_protection_rules/5545303
![image](https://user-images.githubusercontent.com/278814/111518904-e974f880-872c-11eb-9494-57ad70b20be9.png)

Since the tests on Travis are flaky though I'll just remove this line for now.  I think it could be added back after moving tests to Github Actions and checking the right boxes.

Reply:
@albemarle Your PR from 14 months ago mentions that prospectus only auto merges testers and linters if tests pass, do you happen to know how I can make it do that here? 
![image](https://user-images.githubusercontent.com/278814/111521019-0d393e00-872f-11eb-8a3b-3327c58b87d0.png)
https://github.com/edx/prospectus/pull/1430#issue-367589492

Reply:
Pretty sure I figured it out, it's in the Github repo settings, but since this repo isn't using Github Actions the tests don't always pass which means we haven't blocked PRs on tests which means it is riskier to automerge any PRs.  I removed the automerge bits.